<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyJoinWdg</class>
 <widget class="QWidget" name="ApplyJoinWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>229</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>229</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>229</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ApplyJoinWdg</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:rgba(0,0,0,0);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(105,105,105);
font: 12pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>申请上麦</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="layout_close"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="bottomMargin">
          <number>19</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="labApplyInfo">
           <property name="styleSheet">
            <string notr="true">color:rgb(105,105,105);
font: 12pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>申请上麦</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>4</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_TimeNotice">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(105,105,105);
font: 10pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>10s后自动拒绝</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="bottomMargin">
             <number>24</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonAgree">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color: rgb(255, 247, 247);
	border-image: url(:/sysButton/img/sysButton/okButton.png);
}
QPushButton:hover{
	color: rgb(255, 247, 247);
	border-image: url(:/sysButton/img/sysButton/okButtonOn.png);
}
QPushButton:pressed{
	color: rgb(255, 247, 247);
	border-image: url(:/sysButton/img/sysButton/okButton.png);
}
</string>
              </property>
              <property name="text">
               <string>同意</string>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonRefuse">
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
QToolButton{
      background-color:transparent;
      	border-image:url(:/sysButton/img/sysButton/cancelButton.png);
        color: rgb(200, 200, 200);
      background-color:transparent;
      }
      
      QToolButton:hover{
      background-color:transparent;
      	border-image:url(:/sysButton/img/sysButton/cancelButtonOn.png);
        color: rgb(255, 255, 255);
      }</string>
              </property>
              <property name="text">
               <string>不同意</string>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
